deployer/scripts/upgrade.sh

```diff
@@ -513,6 +514,14 @@ function regenerate_secrets_and_support_objects() {
   generate_support_objects
 }
 
+function remove_triggers() {
+
+  for dc in $(oc get dc -l logging-infra -o jsonpath='{.items[?(@.spec.triggers[*].type)].metadata.name}'); do
+    oc patch dc/$dc -p='{"spec": { "triggers" : [] } }'
+    oc deploy dc/$dc --latest
```

Comment:
come to think of it, is there any reason to redeploy? changing the triggers has no implications for the existing pods.